docs/COMPLETIONS.md

```diff
@@ -0,0 +1,49 @@
+### Shell completions
```

Comment:
Nit: I'd move this file into the `completions` directory and call it `README.md`.